packages/guides/src/Nodes/ProjectNode.php

```diff
@@ -91,17 +91,21 @@ public function getCitationTarget(string $name): CitationTarget|null
         return $this->citationTargets[$name] ?? null;
     }
 
-    public function addLinkTarget(string $anchorName, InternalTarget $target): void
+    public function addLinkTarget(string $anchorName, InternalTarget $target, string $linkType = SectionNode::STD_LABEL): void
```

Comment:
Why is `$linkType` not part of the `$target`? It's an property of the `$target` telling me something about what kind of target it is. This would make the search a bit more complex, however we can solve that by creating a collection class that is responsible for the separation of link types. 

This will make the project node less specific for this use case.